packages/jupyter-ai/jupyter_ai/document_loaders/directory.py

```diff
@@ -126,16 +132,24 @@ def split(path, all_files: bool, splitter):
                 ]
                 filenames = [f for f in filenames if not f[0] == "."]
             filepaths += [Path(os.path.join(dir, filename)) for filename in filenames]
```

Comment:
Lean on pathlib to simplify/improve clarity:
```suggestion
            filepaths += [Path(dir) / filename for filename in filenames]
```

Nit: I find `filepaths.extend()` more explicit and readable than `filepaths +=`.

Reply:
Updated as suggested, thanks!